src/components/code_view.py

```diff
@@ -358,7 +361,33 @@ def __init__(self, master, parent, **kwargs):
         for file in self.file_names:
             self.create_new_tab(file)
             self.bind_esc(editor=self.code_editors[file], file_name=file)
-    
+
+    def quick_run(self):
```

Comment:
consider moving this to `CodeEditor`, andon kasi lahat ng pag run ng program, at that level no need i-check updated source code